checker/checker.js

```diff
@@ -284,7 +284,7 @@ async function set_green_is_adobe_employee (ow, args, membership_org) {
   let reason = `membership in github.com/${membership_org}`;
   if (membership_org === 'magento') {
     company += ' (Magento)';
-    reason = `membership in github.com/magento's employees team`;
+    reason = 'membership in github.com/magento\'s employees team';
```

Comment:
I'd like to keep using backticks for exactly the reason that we don't need to escape quotes. Can we tweak the eslint config to allow for this?

Reply:
Turns out that this is not that easy to configure. I though I could allow escapeing with the following setting:
`'quotes': ['error', 'single', { 'avoidEscape': true }]`
But that still does not allow this:
`` `membership in github.com/magento's employees team` ``
And looking at the eslint issues it seems that this is a deliberate choice on their part.

Another option would be this:
`'quotes': ['error', 'single', { 'allowTemplateLiterals': true }]`
But that would mean that this would also be allowed:
`` `https://adobeioruntime.net/api/v1/web/io-solutions/default` ``
And I personally don't think that would be a nice option.

Your thoughts?

Reply:
The later option, allowing template literals, sounds good to me as I am big fan of them!

Reply:
Indeed  I prefer the explicit `'error'` settings in the eslint config over numbers, otherwise I would have to look it up every time 😄